variscite/variscite-bluetooth

```diff
@@ -96,7 +96,9 @@ bt_found || exit 0
 		enable_bt
 
 		# Get BT MAC address
-		BT_MACADDR=$(get_bt_macaddr)
+		#BT_MACADDR=$(get_bt_macaddr)
+		# Provisional fix to fake max address
+		BT_MACADDR="de:ad:be:ef:ca:fe"
```

Comment:
This should be documented somewhere if it's a provsional fix.

Reply:
We will remove all about variscite wifi/bluetooth in a few days, so I don't think that we need to document this issue